qa/tasks/cephfs/test_snap_schedules.py

```diff
@@ -453,4 +453,157 @@ def test_snap_schedule_with_mgr_restart(self):
         # cleanup
         self.fs_snap_schedule_cmd('remove', path=testdir, snap_schedule='1M')
         self.remove_snapshots(testdir[1:])
-        self.mount_a.run_shell(['rmdir', testdir[1:]])
+        self.mount_a.run_shell(['rmdir', testdir[1:]])    
+
+class TestSnapSchedulesSnapdir(CephFSTestCase):
+    CLIENTS_REQUIRED = 1
+
+    TEST_VOLUME_NAME = 'snap_vol'
+    TEST_DIRECTORY = 'snap_test_dir1'
+
+    # this should be in sync with snap_schedule format
+    SNAPSHOT_TS_FORMAT = '%Y-%m-%d-%H_%M_%S'
+
+    def check_scheduled_snapshot(self, exec_time, timo):
```

Comment:
All these functions are already defined in `TestSnapSchedules`. You can just inherit that or move helper functions to separate class and inherit that class in both `TestSnapSchedules` and `TestSnapSchedulesSnapdir` as done in qa/tasks/cephfs/test_volumes.py

Reply:
Yup, that's better. Thanks!